test/unit/specs/MyComponent.spec.js

```diff
@@ -1,7 +1,7 @@
 import { shallow, createLocalVue } from 'vue-test-utils'
 import Vuex from 'vuex'
-import Modules from '../../../src/components/Modules'
-import module from '../../../src/store/module'
+import MyComponent from '../../../src/components/MyComponent'
+import mymodule from '../../../src/store/mymodule'
```

Comment:
renaming it to 'mymodule' as 'module' was dangerously close to 'modules' which is the expected reserved object key name and could be potentially confusing